server/questions.js

```diff
@@ -45,18 +45,18 @@ exports.addQuestionByType = function(qType, question, callback) {
 	questionToAdd.mtime = currentDate;
 
 	switch (question.type) {
-		case 're':
-			questionToAdd.type = common.questionTypes.REGULAR;
+		case common.questionTypes.REGULAR.value:
+			questionToAdd.type = common.questionTypes.REGULAR.value;
 			break;
 
-		case 'mc':
-			questionToAdd.type = common.questionTypes.MULTIPLECHOICE;
+		case common.questionTypes.MULTIPLECHOICE.value:
+			questionToAdd.type = common.questionTypes.MULTIPLECHOICE.value;
 			questionToAdd.choices = question.choices;
 			break;
 
 		default:
 			//callback('type is not defined', null);
-			questionToAdd.type = common.questionTypes.REGULAR;
+			questionToAdd.type = common.questionTypes.REGULAR.value;
```

Comment:
Honestly I kept this because we had to different values of text and integer to represent the types. Now that we have them unified, we can take this out and put some that represents an error.

Reply:
We can do that when you create the 404 error page